src/Strategy/Rebar3.hs

```diff
@@ -35,10 +35,17 @@ mkProject project =
   DiscoveredProject
     { projectType = "rebar3"
     , projectBuildTargets = mempty
-    , projectDependencyGraph = const $ getDeps project
+    , projectDependencyResults = const $ getDeps project
     , projectPath = rebarDir project
     , projectLicenses = pure []
     }
 
-getDeps :: (Has Exec sig m, Has ReadFS sig m, Has Diagnostics sig m) => RebarProject -> m (Graphing Dependency, GraphBreadth)
-getDeps project = Rebar3Tree.analyze' (rebarDir project)
+getDeps :: (Has Exec sig m, Has ReadFS sig m, Has Diagnostics sig m) => RebarProject -> m DependencyResults
+getDeps project = do
+  (graph, graphBreadth) <- Rebar3Tree.analyze' (rebarDir project)
```

Comment:
[nit] can we move the creation of the `graphBreadth` out of `Rebar3Tree.analyze'`? it's always `Complete`
```suggestion
  graph <- Rebar3Tree.analyze' (rebarDir project)
```

Reply:
If it always complete, then why give developers the option of marking it otherwise?

Reply:
I'm going to move forward and merge. If you have a strong preference to take `graphBreadth` out, I will come back and change it

Reply:
yeah looking back on this it's not important. It just felt weird to have `graphBreadth` be constant here, because we could inline it in the `DependencyResults` creation below

but if we add more tactics here it'll make sense for `Rebar3Tree.analyze'` to return `GraphBreadth`